src/DDTrace/Util/CodeTracer.php

```diff
@@ -53,13 +53,6 @@ public function tracePublicMethod(
                 $isTraceAnalyticsCandidate
             ) {
                 $tracer = GlobalTracer::get();
-                if ($tracer->limited()) {
-                    if ($limitedTracerCallHook) {
-                        $limitedTracerCallHook(func_get_args());
-                    }
-
-                    return dd_trace_forward_call();
-                }
```

Comment:
`CodeTracer` is dead code and references the legacy `dd_trace()` API which is NOOP. We can remove this file and also remove references to it from `/phpstan.ddtrace.neon` and `bridge/_files.php`.

Reply:
done (and also the other suggestions)